vrk-kpa/suomidigi
# SUOPA-789 Groups and communities

Notice! This PR also includes SUOPA-790!

To test: 
- `make fresh drush-updb drush-cr`
- Log in and go to edit the front page. https://suomidigi.docker.sh/node/7/edit
   - Add new Views paragraph with a title "Tule mukaan kehittämään"
        - Select "Nostot: Ryhmät" for the view and "Ruudukko" for the display
        - Copy the texts from https://xd.adobe.com/view/d3c60a1d-1d76-4a8b-4797-030e9c2e717f-a5f7/
   - Save the node and check that the groups look like in the proto
- Go to https://suomidigi.docker.sh/node/159/edit
   - Edit the first View paragraph and select "Ruudukko" for the view display
   - Edit the second View paragraph and select "Ruudukko" for the view display
   - Save the page and check that the group and communities teasers looks like the ones in proto: https://xd.adobe.com/view/d3c60a1d-1d76-4a8b-4797-030e9c2e717f-a5f7/screen/d752c906-b2b9-44f8-98c2-af3b5a250f4e/Suomidigi-2020-ryhmat-verkostot-XL
